### ISSUE TITLE
Replication not working between KNB and LTER metacats

### ISSUE THREAD

**Issue by mbjones** [NCEAS/metacat#649]
---

Author Name: **Michael Daigle** (Michael Daigle)
Original Redmine Issue: 3881, https://projects.ecoinformatics.org/ecoinfo/issues/3881
Original Date: 2009-03-11
Original Assignee: Jing Tao

---

After the 1.9.0 upgrade on knb and lter, docs are not replicating from knb to lter.  Jing is working with Duane C to resolve this issue.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Jing Tao** (Jing Tao)
Original Date: 2009-03-11T20:21:01Z

---

LTER changed the host name of metacat. But the certificate and the host name in knb xml_replication has not been updated. We think this is the reason why replication failed. We are waiting to restart tomcat after changing certificate which Duane just created today.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Jing Tao** (Jing Tao)
Original Date: 2009-03-12T01:07:12Z

---

After getting the new certificate issued for metacat.lternet.edu and change the server to metacat.lternet.edu:8443/knb/metacat/servlet/replication in knb xml_replication, tomcat of knb was restarted. 

I used morpho to insert an new eml 210 doc, then the doc can be searched and read in both knb and lter. I deleted the doc in knb and lter one was gone too.

So i think the replication is working now.
